lib/stateParser.mly

```diff
@@ -179,8 +180,8 @@ maybev_list:
 
 maybev_label:
 | maybev { $1 }
-| PROC COLON NAME { mk_lab $1 $3 }
-| NUM COLON NAME { mk_lab (Misc.string_as_int $1) $3 }
+| PROC COLON NAME { mk_lab ($1, $3) }
```

Comment:
It seems possible to restore one deleted line without having a conflict:
```suggestion
|  NUM COLON NAME { mk_lab (Misc.string_as_int $1) $3 }
| PROC COLON NAME { mk_lab ($1, $3) }
```

Reply:
That way we'd restore previous behaviour to accept initialisations such as `0:X1=0:L0`.